src/types.ts

```diff
@@ -90,6 +90,7 @@ export interface ErrorNonJson extends Error {
 
 export interface Options extends https.RequestOptions {
     agent?: https.Agent;
+    header?: {};
```

Comment:
Can you please double-check if it's `header` or `headerS`? I was under impression old code used plural `headers` everywhere, and so does [one of your changes](https://github.com/dgraph-io/dgraph-js-http/pull/32/files?file-filters%5B%5D=.ts#diff-5b74f88b6c4904282773b2762a2d3602R382)

Reply:
I will change it to "headers" in the Options interface. 

Do you also want me to make the same change [here](https://github.com/dgraph-io/dgraph-js-http/pull/32/files?file-filters%5B%5D=.ts#diff-5b74f88b6c4904282773b2762a2d3602R382)? It made sense to name it "header" as the purpose of this method is to insert a singular header in the options.headers object.

Reply:
setCustomHeader sets one header, so its name is OK.
options.headers keeps [0..N] different headers so plural is better in that context.
that's the logic I suggest we follow wrt to all names